lib/ShareProvider.js

```diff
@@ -0,0 +1,466 @@
+    "name": "LinkedIn",
+    "origin": "https://www.linkedin.com",
+    "pageSize": {
+      "share": {
+        "height": 570,
+        "width": 520
+      }
+    },
+    "shareURL": "https://www.linkedin.com/shareArticle?mini=true&url=%{url}&title=%{title}&trk=FIREFOX_ACTIVATION",
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/linkedin.html",
+    "updateDate": Date.now(),
+    "installDate": Date.now()
+  },
+  "social.manifest.www-tumblr.com": {
+    "author": "Tumblr, Inc.",
+    "description": "Post text, photos, quotes, links, music, and videos from your browser, phone, desktop, email or wherever you happen to be. You can customize everything, from colors to your theme's HTML.",
+    "homepageURL": "https://www.tumblr.com",
+    "icon32URL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAOtJREFUeNpi/P//P8NAAiaGAQajDmAhpMDCPY5sw0/sXES5A/AAQST2e5qFAA6gDcRXkPiM9E4DFgOdCJMG0gEgy63olgvQQBoQTyaQIElKlKQ44CwQG+GQe4fGZ6RFFBiNzJIQCQjhCO5PQKxADwfgSlj/KSkJR2vDUQeQ44A/aHxuSqp1chzwBEtOikZzEE0dsAmL2AIgvg/ET4F4Ha0d0AXEb7GIgwojKSB2BGIRWjoA5EsvIH6OQ54ViO1p3SQ7BcRq0OrZHYjlgfgbtJm2Goi30NoBIPAFiPugmGzAONo1G3XAQDsAIMAAyz0m+77iTBQAAAAASUVORK5CYII=",
+    "icon64URL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAaJJREFUeNrsmzFLAzEUxxMp4lQEHR0ERRCKdFBwdBDq4CAoLpV2cPAjuBRxUcRR/AS6OwiCIq5SdFdc66R0KYKgIvEfmqGIYLTmztz7P/htL3fpj8u7d7mrNsYoydGjhAcFUAAFUAAFUAAFUAAFUIDQyP31AadLleCTrp8dRH0F3APzBSKWQD8YklwDxqUXwUnpAuYlCxgBs5IFbP7HviOpCS2BFamd4AI4lNgK58EeOAJ9YlphFztgzTU+vvFdN6hjErDOp0EKoAAKiCFCFUH9wwofrMrzCqAACqAACqAACqAACqCAdAU8eeQMZ1lAwyOnnGUB1x45G6q9c9w5p15QAoOxCzjxyLE/1u4eP4IrcOeWzqkKuKWelIBj0PTMHQBTYMxJsVGNXcAr2O5ifBFMxH4btO8ILroYX4ldwDtYBPVfjl/OQiPUAjNg1y0L37gBq1npBF9U+8XJKKiBS/D8KecN3IJ9MAcK4DzEZHIqvbAfS205tKv+eSfjQSX04VSaAjrDuNtkM+kTa/5pik+DFEABFEABFEABFEABMuNDgAEAKzQ9WAq/3UEAAAAASUVORK5CYII=",
+    "iconURL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAJNJREFUeNpi/P//PwMlgAWZY+EeR5SmEzsXwdlMeNQJAfFMKMYJ8BkQCcRpUEyyAc5A3ElyGECBCRDvBmJGJDFYSDMS4wI+bApJ8cI+LAYw4jKUiYFCQFMDvqKlCW5SDdiDxH4LxIdJNaAIiK8h8ZmBmIuYdAAD94BYB4hVgfg3EN8nNiExoCWgW/gUMFKanQECDABkohjLM1oQUAAAAABJRU5ErkJggg==",
+    "name": "Tumblr",
+    "origin": "https://www.tumblr.com",
+    "pageSize": {
+      "share": {
+        "height": 430,
+        "width": 550
+      }
+    },
+    "shareURL": "https://www.tumblr.com/share?v=3&u=%{url}&t=%{title}&s=%{description}",
+    "version": 1,
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/tumblr.html",
+    "updateDate": Date.now(),
```

Comment:
just confirming, this is supposed to be the date/time AS was supposed to be installed?